lib/3scale/backend/user.rb

```diff
@@ -142,6 +161,17 @@ def self.clear_cache(service_id, user_id)
         Memoizer.clear key
       end
 
+      def self.delete_users_in_batch(service_id, usernames)
```

Comment:
Methods defined under `private` with `self.f` are not private in Ruby. Just one of those things that might be confusing at first :)

Private class methods need to be defined under `private` inside `class << self` or using https://ruby-doc.org/core-2.5.3/Module.html#method-i-private_class_method

No need to change this now, as I see that there are other methods in this class that are supposed to be private but are not.